doc/whatsnew/2.12.rst

```diff
@@ -3,11 +3,25 @@
 ***************************
 
 :Release: 2.12
-:Date: TBA
+:Date: 2021-11-14
 
 Summary -- Release highlights
 =============================
 
+In 2.12, we introduced a new option ``py-version`` that will permits to analyse code for a python
+version that may differ from your python interpreter. This is not effective for all checkers but
+permits for example to check for python 3.5 code smell (f-string used) using pylint with python 3.6.
+The minimum version to run pylint is now 3.6.2, while the last working version for python 3.6.0
+and 3.6.1 was pylint 2.9.3.
+
+On top of fixing a lot of false positives and bug, we also added new default checks, like
+``use-implicit-booleaness-not-comparison``, ``overridden-final-method``, or ``useless-with-lock``.
+There's also better check for TOML configurations.
+
+The checker for Yoda conditions is now an extension, you might want to enable it if you were
+relying on this check. There's also a new extension checker, ``consider-using-any-or-all`` that
+detect for loops that could be replaced by any or all, entirely contributed by @areveny,
```

Comment:
@areveny are you ok with being highlighted in the release highlight ?

Reply:
```suggestion
detects for loops that could be replaced by any or all, entirely contributed by @areveny,
```

Reply:
Sure! I appreciate the shout out.